app/views/tasks/show.html.erb

```diff
@@ -31,6 +32,46 @@
   <p>(<%= @task.latitude %>, <%= @task.longitude %>)</p>
 <% end %>
 
+<% if @task.latitude.present? and @task.longitude.present? %>
+  <div id="map" class="map" style="height:250px"></div>
+  <div style="display: none;">
+  <a class="overlay" id="task" target="_blank" href=""><%= @task.title %></a>
```

Comment:
@marshall-alex class "overlay" does not appear to have a purpose as of now. Do you have one for this?

Reply:
No, sorry, I should've removed it. I ended up using the id instead and forgot to take out the class